keycloak-iam/src/main/java/com/valtimo/keycloak/service/KeycloakUserManagementService.java

```diff
@@ -163,6 +166,17 @@ public List<ManageableUser> findByRoles(SearchByUserGroupsCriteria groupsCriteri
             .collect(Collectors.toList());
     }
 
+    @Override
+    public ManageableUser getCurrentUser() {
```

Comment:
This probably works, but why don't we find the user by id? I'm not sure if a user always has an email, but they should always have an id.

Reply:
Because this is for keycloak. Keycloak doesnt have a user ID. Keycloak puts only the email into the Authentication object. See:
https://github.com/valtimo-platform/valtimo-backend-libraries/blob/development/keycloak-iam/src/main/java/com/valtimo/keycloak/security/jwt/authentication/KeycloakTokenAuthenticator.java#L83

Reply:
Not sure that's the best idea, but for now it's fine then.